server/server.js

```diff
@@ -27,7 +27,8 @@ app.get('/test', db.getTest);
 app.post('/test', db.postTest);
 
 // actual routes TBD
-app.post('/submit', validationMiddleware.validSubmitRequestProperties, upload.single('file'), story.createStory);
+// app.post('/submit', validationMiddleware.validSubmitRequestProperties, upload.single('file'), story.createStory);
```

Comment:
Currently an issue with validation multer form data if you could take a look @XinyangZhu. Running with validation always returns invalid parameters.